tasks/enforced-immutability/src/main.rs

```diff
@@ -2,9 +2,9 @@
 
 fn main() {
     // Rust let bindings are immutable by default.
-    let x = 3;
+    let z = 3;
     // This will raise a compiler error:
-    // x += 2;  //~ ERROR cannot assign to immutable borrowed content `*y`
+    // z += 2;  //~ ERROR cannot assign to immutable borrowed content `*y`
```

Comment:
Is this error message right? `y` isn't borrowed yet.

Reply:
I think it was not right in first place already. I will update the error message